robottelo/rerun_failures/uncollector.py

```diff
@@ -0,0 +1,111 @@
+import logging
```

Comment:
I think we should structure the location of this plugin a bit more generically, instead of creating `rerun_failures` directory that likely won't have any other content in it.

How about creating a more generic `robottelo/pytest_plugins` directory, where we can start adding modules, and moving some existing plugins from individual conftest.py modules into a common location.

We can split fixtures and markers into their own directories as well for some additional organization.

Reply:
Yeah !
And, you have answered to my question in description :
```
Maybe different plugin name than uncollector. If the reviewer demands?
```
I was also not in favor of adding it there but I saw bz(issue_handelers) module there.

I definately agree with you on creating a separate package for plugins !

Reply:
Still thinking about 'uncollector' naming :)

We _could_ include this hook and the ReportPortal module all in one, but I think there's some value to keeping them split and finding a good generic name for the plugin.

Although the behavior is un-collection, how about naming the module `failure_collection.py`?

Reply:
I would like to be more specific `portal_collection`, cause we might have failure collections later but not based on report portal.

or 

analyzed_collection ?